lms/djangoapps/instructor_task/tests/test_integration.py

```diff
@@ -569,3 +569,16 @@ def test_one_group_problem(self):
                 ],
                 ignore_other_columns=True
             )
+
+
+class TestGradeReportConditionalContent(TestReportMixin, TestConditionalContentSequential, TestIntegrationTask, GradeReportConditionalContentMixin):
+    """
+    Tests based on TestConditionalContentSequentialMixin
+    """
+
+
+class TestGradeReportConditionalContentVerticals(TestReportMixin, TestConditionalContentVerticals, TestIntegrationTask, GradeReportConditionalContentMixin):
+    """
+    Tests based on TestConditionalContentVerticalsMixin
```

Comment:
> Please point me to the code which uses this test classes.
 The same as in TestGradeReportConditionalContent.

"""This tests are part of openedx.core.djangoapps.util.testing.TestConditionalContentSequential and 
openedx.core.djangoapps.util.testing.TestConditionalContentVertical Mixins."""